src/util/options.rs

```diff
@@ -426,6 +426,165 @@ impl Options {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum GCTriggerSelector {
+    FixedHeapSize(usize),
+    DynamicHeapSize(usize, usize),
+    Delegated,
+}
+
+impl GCTriggerSelector {
+    const K: usize = 1024;
+    const M: usize = 1024 * 1024;
+    const G: usize = 1024 * 1024 * 1024;
```

Comment:
If we allow suffixes, I suggest we also add support for "T" (and maybe "P", and "E" as well).   1TB of RAM is quite possible for today's servers.  x86_64 doesn't really support more than 256TB (`1<<48`) of memory, so 1PB cannot be supported easily.  But it is still nice to have, just for consistency.

Reply:
I added `T` but not `P`. It seems unlikely that we will use petabytes of RAM any time soon. Plus they can use `1024T` anyway.